code/modules/food_and_drinks/food/foods/misc_food.dm

```diff
@@ -63,33 +63,101 @@
 //		Salads		//
 //////////////////////
 
-/obj/item/reagent_containers/food/snacks/aesirsalad
-	name = "aesir salad"
-	desc = "Probably too incredible for mortal men to fully enjoy."
-	icon_state = "aesirsalad"
+/obj/item/reagent_containers/food/snacks/salad
+	name = "salad"
+	desc = "Should not spawn or be possible to be created. Contact a coder if this somehow exists."
+	icon = 'icons/obj/food/soupsalad.dmi'
+	icon_state = "salad"
 	trash = /obj/item/trash/snack_bowl
 	filling_color = "#468C00"
 	bitesize = 3
+	list_reagents = list("nutriment" = 6,"vitamin" = 4)
+	tastes = list("lettuce" = 2)
```

Comment:
Why would a lone thing in a weighted list have a weight of 2? Could probably make this a normal list

Reply:
As commented on the Discord; every other item I can see for the most part does it this way. I presume it's for custom food weighting I.E you cram it into a sandwich. It's consistent with other food.